web3/main.py

```diff
@@ -245,3 +252,6 @@ def enable_unstable_package_management_api(self):
         from web3.pm import PM
         if not hasattr(self, '_pm'):
             PM.attach(self, '_pm')
+
+    def enable_strict_bytes_type_checking(self):
+        return setattr(self, 'codec', ABICodec(build_strict_registry()))
```

Comment:
any reason this isn't just `self.codec = ABICodec(...)`?

Reply:
Nope! 👍